src/Components/GrantsShow.js

```diff
@@ -176,110 +183,131 @@ export default function GrantsShow(props) {
-            <Button onClick={handleShowGrantEditModal}>Edit</Button>
-            <Modal
-              onClose={handleCloseGrantEditModal}
-              show={showGrantEditModal}
-            >
-              <Card>
-                <Card.Body>
-                  <GrantEdit
-                    grant={grant}
-                    onSubmit={handleCloseGrantEditModal}
-                    onCancel={handleCancelGrantEdit}
-                  />
-                </Card.Body>
-              </Card>
-            </Modal>
-            <Modal
-              className="modal-popup"
-              onClose={handleCloseGrantCopyModal}
-              show={showGrantCopyModal}
-            >
-              <Card>
-                <Card.Body>
-                  <GrantCopy grant={grant} />
-                </Card.Body>
-              </Card>
-            </Modal>
-          </div>
-        </header>
-        <dl className="GrantsShow__Fields">
-          <div className="GrantsShow__Deadline">
-            <dt>Deadline:&nbsp;</dt>
-            <dd>{formatDate(grant.deadline)}</dd>
-          </div>
-          <dt>Funding Organization</dt>
-          <dd>{grant.funding_org_name}</dd>
-          <dt>Purpose</dt>
-          <dd>{grant.purpose}</dd>
-          <dt>RFP URL</dt>
-          <dd>{grant.rfp_url}</dd>
-          <dt>Total word count:</dt>
-          <dd>{totalWordCount}</dd>
-        </dl>
-      </section>
+        <section className="GrantsShow__Overview">
+          <header className="GrantsShow__Header">
+            <h1 className="GrantsShow__Title">{grant.title}</h1>
+            <div className="GrantsShow__Actions">
+              <Button>
+                {" "}
+                <Link
+                  to={`/organizations/${currentOrganizationId}/grants/${grant.id}/copy/`}
+                >
+                  Copy
+                </Link>
+              </Button>
+              <Button>
+                {" "}
+                <Link
+                  to={`/organizations/${currentOrganizationId}/grants/${grant.id}/edit/`}
+                >
+                  Edit
+                </Link>
+              </Button>
+              {/* <Modal
+                onClose={handleCloseGrantEditModal}
+                show={showGrantEditModal}
+              >
+                <Card>
+                  <Card.Body>
+                    <GrantEdit
+                      grant={grant}
+                      onSubmit={handleCloseGrantEditModal}
+                      onCancel={handleCancelGrantEdit}
+                    />
+                  </Card.Body>
+                </Card>
+              </Modal>
+              <Modal
+                className="modal-popup"
+                onClose={handleCloseGrantCopyModal}
+                show={showGrantCopyModal}
+              >
+                <Card>
+                  <Card.Body>
+                    <GrantCopy grant={grant} />
+                  </Card.Body>
+                </Card>
+              </Modal> */}
```

Comment:
we should circleback / make a ticket to fix the functionality and styling of the grant edit and grant copy pages